portfolio/src/main/webapp/index.html

```diff
@@ -20,9 +20,25 @@
       please see the <a href="gallery.html">gallery page</a> of this
       portfolio
     </p>
+    <!-- select element allowing the user to specify a max
+        number of comments to display on the page -->
+    <select type="range" min="0" max="25"
+        id="maxComments"
+        onchange="selectionChangeHandler(this.value);"
+    >
+      <option value="5">5</option>
+      <option value="10">10</option>
+      <option value="25">25</option>
+      <option value="50">50</option>
```

Comment:
The range above says max=25 but there is an option for 50 here?

Reply:
I've fixed this now 👍 decided to omit 50 since I can't see a use case for 50 that isn't already covered by 'All' https://github.com/thomasogara/step-repository/pull/28/commits/1d37ae9d67c7098ed6d344ed4bc0042c900b4399